eng/MPack.targets

```diff
@@ -75,4 +75,14 @@
     </ItemGroup>
   </Target>
 
+  <!--This Target is used when this AddIn is built side-by-side with monodevelop folder so it "injects" itself as AddIn instead of downloaded AddIn-->
+  <Target Name="RegisterAddin" AfterTargets="Build" Condition="Exists ('$(_MonoDevelopPath)')" >
```

Comment:
I agree with @dougbu and @rynowak. 

It would be valuable to have this target but have it be aware of a few MSBuild properties that could be set from the command line build aka `./build /p:_MonoDevelopPath=...` etc. If that's something you're interested could you add some comments here stating how this target could be called from the command line in order to enable overriding the monodevelop addin?

Reply:
nit: `/p:MonoDevelopPath=...` (don't make it necessary for users to override what appears to be an internal property)